authors/apps/articles/views.py

```diff
@@ -70,4 +74,33 @@ def delete(self, request, slug, format=None):
         self.check_object_permissions(request, article)
         article.delete()
         return Response(status=status.HTTP_204_NO_CONTENT)
-    
+
+
+class LikesView(APIView):
+    model = None    # Data Model - Articles or Comments
```

Comment:
use docstring to give info on what the class is about, and all other methods you write

Reply:
I have added the docstrings